claims_hosp/setup.py

```diff
@@ -13,6 +13,7 @@
     "pylint==2.8.3",
     "pytest-cov",
     "pytest",
+    "cvxpy<=1.4",
```

Comment:
from the [cvxpy update notes](https://www.cvxpy.org/updates/index.html#:~:text=In%201.5%2C%20CVXPY%20plans%20to%20start%20using%20Clarabel%20instead%20of%20ECOS%20by%20default%20for%20some%20categories%20of%20problems.%20In%201.6%2C%20we%20plan%20to%20no%20longer%20install%20ECOS%20as%20a%20CVXPY%20dependency.)
```suggestion
    "cvxpy<1.6",
```